src/Surfnet/ServiceProviderDashboard/Infrastructure/Jira/Repository/DevelopmentIssueRepository.php

```diff
@@ -56,6 +68,16 @@ public function findByManageIds(array $manageIds)
         return new IssueCollection($result);
     }
 
+    public function createJiraTicket(
+        ManageEntity $entity,
+        PublishProductionCommandInterface $command,
+        string $issueType,
+        string $summaryTranslationKey,
+        string $descriptionTranslationKey
+    ): Issue {
+        return new Issue('KEY-27', 'fake-type', Issue::STATUS_OPEN);
```

Comment:
Just for my understanding. What does KEY-27 mean? Perhaps, use a more explaining constant?

Reply:
Jira makes a human readable ID for each ticket. SURF-9 for example. I just created a simple look alike here. Given this is a very explicit dev-only test class. I do not see the need to extract this into a fancy constant.